packages/wizard/src/wizard-step.tsx

```diff
@@ -0,0 +1,206 @@
+import { useColorModeValue } from "@chakra-ui/color-mode"
+import { CheckIcon, CloseIcon } from "@chakra-ui/icons"
+import { Spinner } from "@chakra-ui/spinner"
+import {
+  chakra,
+  forwardRef,
+  HTMLChakraProps,
+  ThemingProps,
+  useStyles,
+} from "@chakra-ui/system"
+import { darken, lighten, mode } from "@chakra-ui/theme-tools"
+import { Collapse } from "@chakra-ui/transition"
+import { AnimatePresence, motion } from "framer-motion"
+import * as React from "react"
+import { WizardConnector } from "./wizard-connector"
+
+const AnimatedCheck = motion(CheckIcon)
+const AnimatedCloseIcon = motion(CloseIcon)
+const AnimatedSpan = motion(chakra.span)
+export interface WizardStepProps extends HTMLChakraProps<"div"> {
```

Comment:
I thing here is the biggest difference from my initial idea about a Stepper component. In your proposal your are only allowed to have a text (description) as content for your Step ( WizardStep). This is in my opinion not flexible enough.

Reply:
So the `description` prop would only be for a "sub title" to the actual step title. How I have it currently is that the user can pass child components to the `Step` component which would render as step content (only in vertical orientation, though). I would be happy to make it more explicit like how you have it in the pro version api and export a `StepContent` component which would handle the collapsing of content when the user switches between steps